lib/instrumenter/collectors-vm/link.js

```diff
@@ -45,13 +46,42 @@ var LINK_KSTATS = [
     }
 ];
 
+/*
+ * Given an insstance uuid, get nics info from vminfod
+ */
```

Comment:
This should call out that we're using an unstable interface (i.e. vminfod local HTTP API) and the reason for it is that forking out and calling `vmadm` would add considerable latency to a somewhat latency sensitive agent.

Reply:
The comment has been added.